packages/fleet/lib/supervisor.ts

```diff
@@ -271,9 +318,29 @@ export class Supervisor {
         return nodes.remove(this.dbClient.db, { nodeId: node.id });
     }
 
-    private async warnNode({ node }: { type: 'WARN'; node: Node }): Promise<Result<Node>> {
+    private async finishingTimeout({ node }: { type: 'FINISHING_TIMEOUT'; node: Node }): Promise<Result<Node>> {
+        // Locally we assume the node is IDLE
+        // since the process is likely already killed
+        // and the node is not able to notify back that it is idle
+        if (isLocal) {
```

Comment:
I would love to have a dedicated env var instead of using isLocal

Reply:
actually we have a RUNNER_TYPE env vars. Let's use this